lineage/figures/figure11.py

```diff
@@ -0,0 +1,80 @@
+""" This file depicts the distribution of phase lengths versus the states. """
+import numpy as np
+import pandas as pd
+from matplotlib.ticker import MaxNLocator
+import seaborn as sns
+
+from ..Analyze import run_Analyze_over
+from ..data.Lineage_collections import Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM, Lapatinib_Control, Lapt25uM, Lapt50uM, Lap250uM
+from .figureCommon import getSetup, subplotLabel
+
+data = [Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM, Lapatinib_Control, Lapt25uM, Lapt50uM, Lap250uM]
+
+lpt_g1 = []
+lpt_g2 = []
+lpt12_g1 = [] # for fig. 12
+lpt12_g2 = [] # for fig. 12
+
+gmc_g1 = []
+gmc_g2 = []
+gmc12_g1 = [] # for fig. 12
+gmc12_g2 = [] # for fig. 12
+
+# Run fitting
+output = run_Analyze_over(data, np.repeat([4, 3], 4))
+gemc_tHMMobj_list = [oo[0] for oo in output[0: 4]]
+gemc_states_list = [oo[1] for oo in output[0: 4]]
+lapt_tHMMobj_list = [oo[0] for oo in output[4: 8]]
+lapt_states_list = [oo[1] for oo in output[4: 8]]
+
```

Comment:
There must be something wrong in these lines. I stared at them for more than half an hour but don't see anything wrong with it. All four populations for gemcitabine and all 4 populations in lapatinib are the same (as you said yersterday)

Reply:
`output`'s length is 8 (which is correct), and each `output[i]` has the length of 3 (thmm_obj, states, likelihood).

Reply:
Have you printed the states lists and checked whether they are different? If they are the problem is after this point. If not it's before.

Reply:
I think the screenshot below, shows the problem is before this point, am I right?

Reply:
Well, before the point you printed. There are multiple steps between there and here.